internal/mailer/mailer.go

```diff
@@ -2,12 +2,12 @@ package mailer
 
 import (
 	"net/url"
-
 	"github.com/netlify/mailme"
 	"github.com/sirupsen/logrus"
 	"github.com/supabase/gotrue/internal/conf"
 	"github.com/supabase/gotrue/internal/models"
 	"gopkg.in/gomail.v2"
+	"github.com/google/uuid"
```

Comment:
Actually, we use this package for UUIDs https://github.com/gofrs/uuid but this is great I'll fix it up!